.gitignore

```diff
@@ -7,3 +7,5 @@ node_modules
 .env
 *.config.json
 deploy.sh
+
+package-lock.json
```

Comment:
Do not leave this file out, you're supposed to commit it

Reply:
It wasnt there before though ? Do you still want it ?

Reply:
Yes